python/dgl/backend/pytorch/util.py

```diff
@@ -0,0 +1,15 @@
+from .tensor import attach_grad, is_recording, data_type_dict, zerocopy_to_dlpack, zerocopy_from_dlpack
+from ..._ffi.ndarray import empty_shared_mem
+
+dtype_dict = data_type_dict()
+dtype_dict = {dtype_dict[key]:key for key in dtype_dict}
+
+def get_shared_mem_array(name, shape, dtype):
+    new_arr = empty_shared_mem(name, False, shape, dtype_dict[dtype])
+    dlpack = new_arr.to_dlpack()
+    return zerocopy_from_dlpack(dlpack)
```

Comment:
these are not specific to pytorch. can we implement them somewhere else?

Reply:
Do we have a good place to put these code? It is also used in DistDGL module.